test/cases/api.sh

```diff
@@ -330,6 +330,43 @@ EndOfMessage
 fi
 export SUBSCRIPTION_BLOCK
 
+# Define the customizations for the images here to not have to repeat them
+# in every image-type specific file.
+case "${IMAGE_TYPE}" in
+  # The Directories and Files customization is not supported for these image types.
+  "$IMAGE_TYPE_EDGE_COMMIT"|"$IMAGE_TYPE_IOT_COMMIT"|"$IMAGE_TYPE_EDGE_INSTALLER"|"$IMAGE_TYPE_IMAGE_INSTALLER")
```

Comment:
Didn't we decide we're supporting commits but not `edge-installer`, `edge-raw-image`, and `edge-simplified-installer`?
I see we're testing it in the ostree tests, so is this just for the API?

Also the image installer should be okay to support.  But maybe we can add that later separately.

Reply:
> Didn't we decide we're supporting commits but not `edge-installer`, `edge-raw-image`, and `edge-simplified-installer`?

Yes.

>  I see we're testing it in the ostree tests, so is this just for the API?

So I ended up not focusing on testing the commit image in the API test, since it is tested in its own test case. But thinking about this, it should be possible to inspect the custom files in the commit using `ostree`.

Also there is probably no good reason why should I not at least enable the customization for the commit image type.

> Also the image installer should be okay to support. But maybe we can add that later separately.

I already have the support for the `raw-image` and `simplified-installer` implemented on another branch. I removed all of it from this PR, because the decision is to go with just the commit and container image types for 8.8 / 9.2 which this PR targets.

Reply:
I added the customization and test for ostree commit in API test.

Reply:
> > Also the image installer should be okay to support. But maybe we can add that later separately.

> I already have the support for the raw-image and simplified-installer implemented on another branch. I removed all of it from this PR, because the decision is to go with just the commit and container image types for 8.8 / 9.2 which this PR targets.

I mean the non-ostree Anaconda installer that installs a traditional system (`image-installer`).

Reply:
We should instead test it in https://github.com/osbuild/osbuild-composer/blob/main/test/cases/installers.sh in a follow-up.